python/openchange/provision.py

```diff
@@ -797,36 +824,32 @@ def newgroup(names, lp, creds, groupname, mail=None,):
 add: msExchRecipientDisplayType
 msExchRecipientDisplayType: %(recipient_display_type)s
 """
-        ldif_value = extended_group % {"group_dn": group_dn,
-                                       "groupname": groupname,
-                                       "firstorg": names.firstorg,
-                                       "firstorg_x400": names.firstorg[:16],
-                                       "firstou": names.firstou,
-                                       "firstou_x400": names.firstou[:64],
-                                       "smtp_user": smtp_user,
-                                       "mail_domain": mail_domain,
-                                       "recipient_type_details": recipient_type_details,
-                                       "recipient_display_type": recipient_display_type}
-        db.modify_ldif(ldif_value)
-
-        res = db.search(base=group_dn, scope=SCOPE_BASE, attrs=["*"])
-        if len(res) == 1:
-            record = res[0]
-        else:
-            raise Exception("this should never happen as we just modified the record...")
-        record_keys = [x.lower() for x in record.keys()]
+    ldif_value = extended_group % {"group_dn": group_dn,
+                                   "groupname": groupname,
+                                   "firstorg": names.firstorg,
+                                   "firstorg_x400": names.firstorg[:16],
+                                   "firstou": names.firstou,
+                                   "firstou_x400": names.firstou[:64],
+                                   "smtp_user": smtp_user,
+                                   "mail_domain": mail_domain,
+                                   "recipient_type_details": recipient_type_details,
+                                   "recipient_display_type": recipient_display_type}
+    db.modify_ldif(ldif_value)
+
+    res = db.search(base=group_dn, scope=SCOPE_BASE, attrs=["*"])
+    if len(res) == 1:
+        record = res[0]
+    else:
+        raise Exception("this should never happen as we just modified the record...")
```

Comment:
Please, don't put this kind of message, because it **WILL** happen and you know it.

Either put a real error or remove it.